### DISCUSSION TITLE
Running individual tasks within a DAG

### DISCUSSION THREAD

**Discussion by arjunanan6** [apache/airflow#31263] [Q&A] [answered]
### What do you see as an issue?

In #29706, the option to Run tasks were rightfully removed since it was broken. However, I have this question about how to go about running an individual task in a DAG manually, should it be required.

Is there no way to do this other than via the CLI, or am I missing something? 

### Solving the problem

N/A

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment, marked as answer]
Yes. CLI is the only way if you want to do it. And there is no other way.
